src/core/withExperiment.js

```diff
@@ -60,6 +60,18 @@ export const withExperiment = ({
     constructor(props: withExperimentInternalProps) {
       super(props);
 
+      log.info('[WithExperiment.constructor] props:', {
+        _cookie: props._cookie,
+        variantA: props.variantA,
+        variantB: props.variantB,
+      });
+
+      log.info('[WithExperiment.constructor] this.props:', {
```

Comment:
Since this exact component class doesn't use inheritance for anything, I don't see how this log statement would be helpful. It probably won't hurt, though.